### ISSUE TITLE
[Bug] Upload Task, wrong order of paramaters

### ISSUE THREAD

**Issue by bluecurby** [cobbr/Covenant#174]
Hi, first of all: GREAT WORK!!! :)

In the Upload.yaml file the order of the paramters `FileContents` and `FilePath` is wrong.

This results in (I added the FilePath and FileContents variable in the Error output:
 ```
Upload /filepath:"C:\test.txt"

System.FormatException: The input is not a valid Base-64 string as it contains a non-base 64 character, more than two padding characters, or an illegal character among the padding characters. 
   at System.Convert.FromBase64_Decode(Char* startInputPtr, Int32 inputLength, Byte* startDestPtr, Int32 destLength)
   at System.Convert.FromBase64CharPtr(Char* inputPtr, Int32 inputLength)
   at System.Convert.FromBase64String(String s)
   at Task.Execute(String FilePath, String FileContents)
FilePath: VGVzdAo=
FileContent: C:\test.txt
 ```

Correct code would be:
 ```C#
public static class Task {
   public static string Execute(string FileContents, string FilePath)
   {
      try
      {
         File.WriteAllBytes(FilePath, Convert.FromBase64String(FileContents));
         return FilePath;
      }
      catch (Exception e) { return e.GetType().FullName + ": " + e.Message + Environment.NewLine + e.StackTrace; }
   }
}
```

Cheers

**Comment by cobbr** [comment]
Strange, this works from the Task tab, but not the upload popup. Thanks for the heads up @bluecurby!

**Comment by cobbr** [comment]
Fixed in bb2cec7a54c64df9a8855109d6b9c5af9bd88c43

Thanks @bluecurby . Let me know if you continue to have problems.
